src/log-store/src/kafka/log_store.rs

```diff
@@ -174,98 +232,82 @@ impl LogStore for KafkaLogStore {
 
         debug!(
             "Built a stream consumer for ns {} to consume entries in range [{}, {}]",
-            ns, start_offset, end_offset
+            provider, start_offset, end_offset
         );
 
         // Key: entry id, Value: the records associated with the entry.
```

Comment:
The key of `entry_records` has changed from `EntryId` to`RegionId`, please also update the doc.

Does this change affect the behavior of read if `entry_records` buffers region id instead of entry id?

Reply:
The original design should be `HashMap<(RegionId, EntryId), Vec<Record>>.` However, I think `RegionId` is the correct choice. I don't want to rely on the `EntryId`. The `EntryId` isn't the real `EntryId` or `Offset`(Just a logical sequence, monotonically increasing, no duplicate within the region)